digos-ambassador/Content/Transformations/Species/snake/Vagina.yml

```diff
@@ -0,0 +1,9 @@
+part: Vagina
+species: snake
+description: "Transforms the target's vagina into that of an anthro shark."
+default_base_colour: "soft pink"
+is_nsfw: true
+shift_message: "{@target}'s slit distends and morphs as it changes shape both internally and externally. The slit becomes tight and welcoming, with protective exterior scales shifting into place to hide soft secondary folds and wet vagina."
```

Comment:
Could already have been tight, welcoming, etc - generalize more.

"hide soft secondary folds and wet vagina" - missing word or confused wording? "vagina" is also a fairly dry, technical term - alternatives?